dev/com.ibm.ws.rest.handler.validator/src/com/ibm/ws/rest/handler/validator/internal/ValidatorRESTHandler.java

```diff
@@ -308,4 +313,16 @@ private JSONObject toJSONObjectForThrowable(Map<String, ?> errorInfo, Throwable
 
         return json;
     }
+
+    @Trivial
+    private String resolvePotentialVariable(String value) {
+        if (value == null || !value.startsWith("${") || !value.endsWith("}"))
```

Comment:
Rather than us owning the logic to determine what is a valid variable and what is not, it would be nice to keep that logic in the variable registry itself.  Can we just send non-null values to VariableRegistry regardless?